code/obj/item/plants.dm

```diff
@@ -213,6 +213,11 @@
 	desc = "A bland but healthy cereal crop. Good source of fiber."
 	icon_state = "oat"
 
+/obj/item/plant/oat/salt
+	name = "oat"
```

Comment:
```suggestion
	name = "salted oat"
```

Reply:
thanks you, for pointing all that out! have a nice day.